transform/snowflake-dbt/models/hightouch/blapi/contact_us_campaign_fact.sql

```diff
@@ -15,7 +15,7 @@ with existing_members as (
         end as campaign_type,
         row_number() over (partition by campaignmember.email order by createddate desc) as row_num
     from {{ ref('campaignmember') }}
-    where campaignmember.campaignid = '7013p000001TuhdAAC'
+    where campaignmember.campaignid IN ('7013p000001NkNtAAK', '7013p000001Ttg5AAC')
```

Comment:
can we document somewhere (code, confluence, etc.) what these hard-coded values correspond to?

Reply:
I'll add in confluence a list of "known" campaigns with links to salesforce.